request/executor.go

```diff
@@ -7,44 +7,85 @@ import (
 )
 
 // ExecuteRequest executes an HTTP request based on the specified options.
-func ExecuteRequest(request Request, profileNames []string, variables map[string]string) (*ExecutedRequestResponse, error) {
-	httpRequest, configuredRequest, httpRequestErr := BuildRequest(request, profileNames, variables)
-	if httpRequestErr != nil {
-		return nil, httpRequestErr
-	}
+func ExecuteRequest(request Request, profileNames []string, variables map[string]string) ([]ExecutedRequestResponse, error) {
+	toExecute := make([]Request, 1)
```

Comment:
maybe `requestsToExecute` is more specific? I just find `toExecute` a little vague, even if we are in the `ExecuteRequests` method

Reply:
Agree.